### PR TITLE
Code standards

### PR DESCRIPTION
**Pull request by ddjnw1yu**
Based on Black, Pylint, Mypy, isort

### CODE REVIEW

File: app/pagination.py [hunk 1]
```diff
@@ -1,211 +1,262 @@
...
             dataset_id = ele["experiments"][0]["submitter_id"]
-            if dataset_id not in ordered_dataset:
-                ordered_dataset.append(dataset_id)
-        return ordered_dataset
+            if dataset_id not in ordered_datasets:
+                ordered_datasets.append(dataset_id)
+        return ordered_datasets
 
-    def get_pagination_data(self, item, match_pair, is_public_access_filtered):
+    def gain_pagination_data(self, item, match_pair, is_public_access_filtered):
```
Comment (alan-wu) [inline app/pagination.py]: Is there a reason to rename this to gain? get is a more common term.

File: app/pagination.py [hunk 2]
```diff
@@ -1,211 +1,262 @@
...
             # Replace the dataset if it has a private version
-            for dataset in private_replacement.keys():
-                displayed_dataset[dataset] = private_replacement[dataset][0]
+            for dataset, data in self._handle_thread_fetch(items).items():
+                displayed_dataset[dataset] = data[0]
         return list(displayed_dataset.values())
 
-    def get_pagination_count(self, item):
-        public_access = [Gen3Config.GEN3_PUBLIC_ACCESS]
+    def gain_pagination_count(self, item):
```
Comment (alan-wu) [inline app/pagination.py]: Is there a reason to rename this to gain? get is a more common term.

File: app/pagination.py (hunk 1 above)
Comment (ddjnw1yu) [reply app/pagination.py]: It's been a while after changing it. Can't really remember the reason. I will change it back to "get".

**Review by alan-wu — COMMENTED** [review]
Looks good to me. I think get_ is better than gain_ for those two functions.

**Review by alan-wu — APPROVED** [review]
Works well, I tested it with the front end and everything seems to be working.

**Pull request merged**